modules/wpml/wpml-api.php

```diff
@@ -183,6 +196,136 @@ public function wpml_is_rtl() {
 		return pll_current_language( 'is_rtl' );
 	}
 
+	/**
+	 * Returns the id of the translation group of a translated element.
+	 *
+	 * @since 3.4
+	 *
+	 * @param mixed  $empty_value  Not used.
+	 * @param int    $element_id   The id of the item, post id for posts, term_taxonomy_id for terms.
+	 * @param string $element_type Optional. The type of an element.
+	 * @return int
+	 */
+	public function wpml_element_trid( $empty_value, $element_id, $element_type = 'post_post' ) {
+		if ( 0 === strpos( $element_type, 'tax_' ) ) {
+			$element = get_term_by( 'term_taxonomy_id', $element_id );
+			if ( $element instanceof WP_Term ) {
+				$tr_term = PLL()->model->term->get_object_term( $element->term_id, 'term_translations' );
+			}
+		}
+
+		if ( 0 === strpos( $element_type, 'post_' ) ) {
+			$tr_term = PLL()->model->post->get_object_term( $element_id, 'post_translations' );
+		}
+
+		if ( isset( $tr_term ) && $tr_term instanceof WP_Term ) {
+			return $tr_term->term_id;
+		}
+
+		return 0;
+	}
+
+	/**
+	 * Returns the element translations info using the ID of the translation group.
+	 *
+	 * @since 3.4
+	 *
+	 * @param mixed  $empty_value  Not used.
+	 * @param int    $trid         The ID of the translation group..
+	 * @param string $element_type Optional. The type of an element.
+	 * @return stdClass[]
+	 */
+	public function wpml_get_element_translations( $empty_value, $trid, $element_type = 'post_post' ) {
+		$return = array();
+
+		if ( 0 === strpos( $element_type, 'tax_' ) ) {
+			$translations = PLL()->model->term->get_translations_from_term_id( $trid );
+			if ( empty( $translations ) ) {
+				return array();
+			}
+
+			$original    = min( $translations ); // We suppose that the original is the first term created.
+			$source_lang = array_search( $original, $translations );
+
+			$args = array(
+				'include'    => $translations,
+				'hide_empty' => false,
+			);
+			$_terms = get_terms( $args );
+
+			if ( ! is_array( $_terms ) ) {
+				return array();
+			}
+
+			$terms = array();
+			foreach ( $_terms as $term ) {
+				$terms[ $term->term_id ] = $term;
+			}
+
+			foreach ( $translations as $lang => $term_id ) {
+				if ( empty( $terms[ $term_id ] ) ) {
+					continue;
+				}
+
+				$return[ $lang ] = (object) array(
+					'translation_id'       => '0', // We don't have something equivalent.
```

Comment:
I think this is better:
```suggestion
					'translation_id'       => '0', // We have nothing equivalent.
```